packages/plugin-core/assets/dendron-ws/vault/v100.md

```diff
@@ -0,0 +1,117 @@
+---
+id: q69fyod4ey0a716b69b64fb
+title: Release Notes 0.100.0
+desc: ""
+updated: 1655327146166
+created: 1654447362351
+documentId: 11cbvNLFHBMK4_4XOAAIaV7WpUHhqxgfgY0xrG2Exubw
+revisionId: >-
+  ALm37BWKnOzlGRBLQnqxfd3v6bUF2R9lo04_bddygMFGLt40sErKXnku7Is_vdSH52YwpLM7eHoJWPjLP6xLuA
+---
+
+<html lang="en">
+  <style>
+    body {
+      margin: 0;
+    }
+    .container {
+      text-align: center;
+      background-color: #4da167;
+      padding-top: 30px;
+      padding-bottom: 30px;
+    }
+    #header-block {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      vertical-align: middle;
+    }
+    .inline-header {
+      display: inline-block;
+    }
+    .inline-header > p {
+      color: white;
+      font-size: larger;
+      margin-bottom: 0;
+      margin-left: 30;
+      margin-right: 30;
+    }
+  </style>
+  <body>
+    <div class="container">
+      <div id="header-block">
+        <div class="inline-header">
+          <p><b>
+            Dendron is launching on Product Hunt today! If Dendron has been of
+            use to you, please help us out by leaving feedback on our launch page. 
```

Comment:
TODO: Add real product hunt link; Modify note contents with latest set of edits.